spec/main_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+books = ['dummy'] # add finished books to this array
+
+RSpec.describe 'books' do
+  before(:all) do
+    books.each do |book|
+      # call the bake script for each book
+      output = `USE_LOCAL_KITCHEN=1 ./bake -b #{book} -i spec/books/#{book}/input.xhtml \
+        -o spec/books/#{book}/actual_output.xhtml`
+      puts output
```

Comment:
We can probably not `puts` the output?  Is there a need for it?  If there is a need for it, only do it if some environment variable or flag or rspec verbosity is set somewhere.